gradle/libs.versions.toml

```diff
@@ -8,7 +8,7 @@ accompanist = "0.28.0"
 androidx-lifecycle = "2.5.1"
 androidx-navigation-compose = "2.5.1"
 coil = "2.2.1"
-compose = "1.3.0"
+composeBom = "2023.01.00"
 composeCompiler = "1.3.1"
```

Comment:
Is the compiler in the BOM?

Reply:
https://developer.android.com/jetpack/compose/setup#why_isnt_the_compose_compiler_library_included_in_the_bom